Scripts/Python/grsnWallPython.py

```diff
@@ -571,16 +554,13 @@ def OnNotify(self,state,id,events):
                             clothing = MaleSuit
                         if not wornItem:
                             wornItem = avatar.avatar.getAvatarClothingList()
-                            for item in wornItem:
-                                DefaultColor1.append(avatar.avatar.getTintClothingItem(item[0],1))
-                                DefaultColor2.append(avatar.avatar.getTintClothingItem(item[0],2))
-                        i = 1
-                        for item in clothing[0:]:
+                            DefaultColor1 = {item[1] : avatar.avatar.getTintClothingItem(item[0],1) for item in wornItem}
+                            DefaultColor2 = {item[1] : avatar.avatar.getTintClothingItem(item[0],2) for item in wornItem}
```

Comment:
```suggestion
                            DefaultColor1 = {item[1]: avatar.avatar.getTintClothingItem(item[0],1) for item in wornItem}
                            DefaultColor2 = {item[1]: avatar.avatar.getTintClothingItem(item[0],2) for item in wornItem}
```

Minor formatting fix.